pkg/services/annotations/accesscontrol/models.go

```diff
@@ -0,0 +1,14 @@
+package accesscontrol
+
+// AccessResources contains resources that are used to filter annotations based on RBAC.
+type AccessResources struct {
+	// Dashboards is a map of dashboard UIDs to IDs
+	Dashboards map[string]int64
```

Comment:
question: do we use the UIDs anywhere? Could we have a list of IDs instead?

Reply:
We don't today, this is setting up for a change that does use them.